README.md

```diff
@@ -0,0 +1,31 @@
+gonat [![Build Status](https://drone.lantern.io/api/badges/getlantern/gonat/status.svg)](https://drone.lantern.io/getlantern/gonat)&nbsp;[![GoDoc](https://godoc.org/github.com/getlantern/gonat?status.png)](http://godoc.org/github.com/getlantern/gonat)
+==========
+
+This library only works on Linux.
+
+Dependencies are managed using Go modules. If using a version of Go prior to 1.13, use the environment
+variable `GO111MODULE=on` to enable use of modules.
+
+In order to work, this library needs to be able to open raw sockets and update the conntrack table
+via netlink. You can give the binary the correct capabilities with:
+
+`sudo setcap CAP_NET_RAW,CAP_NET_ADMIN+ep`
+
+This library requires the nf_conntrack module to be installed at runtime.
+
+```
+modprobe nf_conntrack
+modprobe nf_conntrack_ipv4
+```
+
+iptables needs to be configured to drop the outbound RST packets that the kernel would usually create in response to SYN/ACK
+packets responding to our raw TCP connections. We do this only for tcp connections that are already in ESTABLISHED in conntrack.
+The library manually adds these to conntrack since we're using raw sockets.
+
+`sudo iptables -I OUTPUT -p tcp -m conntrack --ctstate ESTABLISHED --ctdir ORIGINAL --tcp-flags RST RST -j DROP`
```

Comment:
Could we add documentation on how to reverse this change?  I can do so.  I'd like to learn more about iptables anyway.

Reply:
Go for it! To reverse the change, just replace the `-I` with `-D`.

Reply:
Ah, well that's easy haha.  Thanks and will do!